### PR TITLE
fix broken multiline parsing

### LINKED ISSUE #631: Going Down: exittext not loaded
Introduced by #620 😅 

It's failing to parse all exittexts here as `GetClusterText()` returns a blank string. Here's what the mapinfo clusters look like:

image

Probably best to revert that PR for now but I'll take a look.

### PR DESCRIPTION
**Pull request by 72hr**
stupid mistake, fixes #631. I added a couple tests for this case.

### CODE REVIEW

File: Tests/Unit/Util/Parser/SimpleParserTest.cs [hunk 1]
```diff
@@ -352,4 +352,22 @@ public void MultiLineCommentInsideQuotesIsIgnored()
...
+
+    // e.g. Going Down
+    [Fact(DisplayName = "Multiline MAPINFO exittext parses correctly")]
+    public void MultiLineExitTextParses()
+    {
+        string data = @"
+exittext ""*RING RING*
+
+HELLO?
+YEAH, I BROKE THE PARSER...""
```
Comment (lemming104) [inline Tests/Unit/Util/Parser/SimpleParserTest.cs:383]: 😆

**Comment by 72hr** [comment]
Something is strange here... the tests were successful before merging, but after merging one of them failed. If I set up a new Ubuntu environment locally and run the tests, they're successful. I'm curious if the tests will succeed if re-run on the dev branch. The test logs on dev and on dev with these 3 commits reverted look the same to me.

**Comment by nstlaurent** [comment]
You are good, it seems that the automap marker thread is getting created in the tests and some synchronization problem happened. The tests aren't supposed to create that so I will see why that's happening.

**Comment by nstlaurent** [comment]
I pushed the fix to dev so the automap marker thread doesn't get created anymore in the tests so we should be good!

**Pull request merged**